tests/console/aide_check.pm

```diff
@@ -50,18 +41,18 @@ EOF
     assert_script_run "cp /var/lib/aide/aide.db.new /var/lib/aide/aide.db";
 
     # Checks the database for added entries
-    validate_script_output "aide --check 2>&1 || true", sub { m/AIDE found NO differences between database and filesystem. Looks okay!!/ && m/Number of entries:(\s+)2/ }, 300;
+    validate_script_output "aide --check 2>&1 || true", sub { m/AIDE found NO differences between database and filesystem. Looks okay!!/ && m/Number of entries:(\s+)1/ }, 300;
 
     assert_script_run "touch /testdir/t2.log";
 
     # Checks the database for added/changed entries
-    validate_script_output "aide --check 2>&1 || true", sub { m/AIDE found differences between database and filesystem/ && m/Added entries:(\s+)1/ && m/Changed entries:(\s+)1/ }, 300;
+    validate_script_output "aide --check 2>&1 || true", sub { m/AIDE found differences between database and filesystem/ && m/Added entries:(\s+)1/ && m/Changed entries:(\s+)0/ }, 300;
 
-    assert_script_run "rm /testdir/t2.log && echo world >> /testdir/t1.log";
+    assert_script_run "rm -f /testdir/t2.log && echo world >> /testdir/t1.log";
```

Comment:
I think using the -f option the rm will always succeed ? So you can use `;` and maybe `script_run` instead of assert